src/node.rs

```diff
@@ -545,6 +545,25 @@ impl<'a> NodeProperty<'a> {
         }
     }
 
+    /// Attempts to parse the property value as a list of [`usize`].
+    pub fn as_usize_list(self, cell_size: usize) -> impl Iterator<Item = usize> + 'a {
```

Comment:
I think perhaps a better name would be `iter_cell_size_values` or something similar to make it clear that it operates off of cell sized chunks

Reply:
I was also thinking of adding another helper for properties that have `prop-encoded-array` values, i.e. `<addr-cell> <size-cell>`.

What do you think about something like:

```rust
pub fn iter_addr_size_cells_list(self, addr_cell: usize, size_cell: usize) -> impl Iterator<Item = (usize, usize)> + 'a {
```